### PR TITLE
Ft current products

### PR DESCRIPTION
**Pull request by dreplica**
Screenshot_20200426-221734_Expo
#### What does this PR do?

it displays current products to be selected by user

#### Description of Task to be completed?

Have the home screen show the following:

1. seperate cards showing an image for the product .
2. Name of the product is displayed.
3. Button to Add product and Leave product

#### How should this be manually tested?

After cloning the repo,
`CD` into it and `RUN`

javascript
git checkout develop

git pull origin develop

yarn install

yarn test

Using an emulator: Launch the app, and you should see the home screen after loading.

### CODE REVIEW

**Comment by ogbodo** [comment]
This is how its displaying on my device.

Also I discovered when I click one any options there, it disappears... Is that on purpose?

Simulator Screen Shot - iPhone Xs - 2020-04-26 at 22 39 40

**Comment by dreplica** [comment]
> This is how its displaying on my device.
> 
> Also I discovered when I click one any options there, it disappears... Is that on purpose?
> 
> Simulator Screen Shot - iPhone Xs - 2020-04-26 at 22 39 40

-the button showing should be that disparity between ios and android. (I dont have an iphone to test the design)

-i did it to disappear just to make it  have an effect (since the  didnt specify).

File: src/screens/regimen_setup/currentProducts/index.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+              paddingLeft: 0,
+              paddingTop: 0,
+              paddingBottom: 0,
+              paddingRigth: 0,
+              backgroundColor: colors.BUTTON_LIGHT_COLOR
+            }}
+            textStyle={{
+              width: applyScale(98),
+              height: applyScale(20),
+              color: '#BAB200',
```
Comment (ogbodo) [inline src/screens/regimen_setup/currentProducts/index.tsx]: you are suppose to move this color to the color file

File: src/screens/regimen_setup/currentProducts/style.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components/native';
+import applyScale from '../../../utils/applyScale';
+
+export const Container = styled.View`
```
Comment (ogbodo) [inline src/screens/regimen_setup/currentProducts/style.ts:4]: give this component flex 1 for screen optimisation

**Review by ogbodo — CHANGES_REQUESTED** [review]

File: src/screens/regimen_setup/currentProducts/index.tsx (hunk 1 above)
Comment (dreplica) [reply src/screens/regimen_setup/currentProducts/index.tsx]: thanks boss, I've moved it to theme

File: src/screens/regimen_setup/currentProducts/style.ts (hunk 2 above)
Comment (dreplica) [reply src/screens/regimen_setup/currentProducts/style.ts:4]: Changed it to flex:1 and removed width 100%

File: src/screens/regimen_setup/currentProducts/index.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+import { useThemeContext } from '../../../theme';
+import applyScale from '../../../utils/applyScale';
+import { Container, Image, Content, Title, SubContent } from './style';
+
+interface Iprops {
+  title: string;
+}
+
+interface stateType {
+  display: 'flex' | 'none';
```
Comment (ogbodo) [inline src/screens/regimen_setup/currentProducts/index.tsx]: Please is this a type or a value? Am confused here bro

File: src/screens/regimen_setup/currentProducts/index.tsx [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  added: ``;
+}
+
+export default function CurrentProduct(props: Iprops) {
+  const { colors, fonts } = useThemeContext();
+  const [state, setstate] = useState<'flex' | 'none'>('flex');
```
Comment (ogbodo) [inline src/screens/regimen_setup/currentProducts/index.tsx:14]: here too

**Review by ogbodo — CHANGES_REQUESTED** [review]

File: src/screens/regimen_setup/currentProducts/index.tsx (hunk 3 above)
Comment (dreplica) [reply src/screens/regimen_setup/currentProducts/index.tsx]: is a type to use with style{{display}}, style->display property accepts just that

Comment (dreplica) [reply src/screens/regimen_setup/currentProducts/index.tsx]: but am supposed to delete this one to leave the later

File: src/screens/regimen_setup/currentProducts/index.tsx (hunk 4 above)
Comment (dreplica) [reply src/screens/regimen_setup/currentProducts/index.tsx:14]: Sorry for the duplication, this is the one am using, i declared a type for it earlier

**Review by ogbodo — APPROVED** [review]

**Pull request closed without merging**